docker-compose/cassandra-3.11/docker-compose.yml

```diff
@@ -0,0 +1,75 @@
+version: '2'
+
+services:
+  cassandra-1:
+    image: cassandra:${CASSTAG}
+    networks:
+      - stargate
+    ports:
+      - 9044:9042
+    mem_limit: 2G
+    environment:
+      - HEAP_NEWSIZE=128M
+      - MAX_HEAP_SIZE=1024M
+      - CASSANDRA_SEEDS=cassandra-1
+      - CASSANDRA_CLUSTER_NAME=c3-${CASSTAG}-cluster
+  cassandra-2:
+    image: cassandra:${CASSTAG}
+    networks:
+      - stargate
+    mem_limit: 2G
+    depends_on:
+      - cassandra-1
+    environment:
+      - HEAP_NEWSIZE=128M
+      - MAX_HEAP_SIZE=1024M
+      - CASSANDRA_SEEDS=cassandra-1
+      - CASSANDRA_CLUSTER_NAME=c3-${CASSTAG}-cluster
+  cassandra-3:
+    image: cassandra:${CASSTAG}
+    networks:
+      - stargate
+    mem_limit: 2G
+    depends_on:
+      - cassandra-2
+    environment:
+      - HEAP_NEWSIZE=128M
+      - MAX_HEAP_SIZE=1024M
+      - CASSANDRA_SEEDS=cassandra-1
+      - CASSANDRA_CLUSTER_NAME=c3-${CASSTAG}-cluster
```

Comment:
Not sure why we have one coordinator and three C* nodes, but would it actually make more sense to have two each? Also not sure if we can use `replicas` here, at least for the coordinator?